app/models/merritt/etd.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Merritt
```

Comment:
I had understood we planned to remove this model? the ETDs and ARKs are already stored in Fedora, correct?

Reply:
Merritt::Etd is gone now.